packages/time-select/src/time-select.ts

```diff
@@ -0,0 +1,26 @@
+import dayjs from 'dayjs'
+import customParseFormat from 'dayjs/plugin/customParseFormat'
+import { h } from 'vue'
+import { DEFAULT_FORMATS_TIME } from '@element-plus/time-picker/src/common/constant'
+import Picker from '@element-plus/time-picker/src/common/picker.vue'
+import Panel from './panel.vue'
+dayjs.extend(customParseFormat)
+export default {
+  name: 'ElTimeSelect',
+  props: {
+    itype: {
```

Comment:
typo? itype

Reply:
That's a new product from Apple 😝 Fixed.